templates/addressBook.html

```diff
@@ -1,5 +1,12 @@
 <span class="addressBookName">{{ctrl.addressBook.displayName}}</span>
 <span class="utils">
+	<span class="action" ng-show="ctrl.canExport">
+		<span href="{{ctrl.addressBook.url}}?export"
```

Comment:
Do you have the version > 9.0.2?
It works fine here, I don't understand. What's wrong?
![screen-34-04](https://cloud.githubusercontent.com/assets/14975046/14617107/4c3e4e28-05ad-11e6-9d3e-6ae05d187e1f.png)